frontend/src/services/userService.tsx

```diff
@@ -0,0 +1,11 @@
+import api from '../common/helpers/apiHelper';
+
+export const fetchOrganization = async (userId: string, orgId: string) => {
+  const response = await api.get(`/api/organization/${orgId}/user/${userId}`);
```

Comment:
API methods accept generic, so you always need to set a type of data returned from the server. Like this: `api.get<type>(...)`